File: Flights/ReadModels/Pizza.cs
namespace Pizza.ReadModels
{
    public class Pizza
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string SmallDescription { get; set; }
        public string BigDescription { get; set; }
        public PizzaType Type { get; set; }
        public float Price { get; set; }

        public Pizza(Guid id, string name, string smallDescription, string bigDescription, PizzaType type, float price)
        {
            Id = id;
            Name = name;
            SmallDescription = smallDescription;
            BigDescription = bigDescription;
            Type = type;
            Price = price;
        }
    }


}

File: Flights/ClientApp/src/app/api/models/pizza.ts
/* tslint:disable */
/* eslint-disable */
import { PizzaType } from './pizza-type';
export interface Pizza {
  bigDescription?: null | string;
  id?: string;
  name?: null | string;
  price?: number;
  smallDescription?: null | string;
  type?: PizzaType;
}
